src/main/java/com/feedzai/commons/sql/abstraction/engine/impl/H2Engine.java

```diff
@@ -56,6 +58,10 @@
  * @since 2.0.0
  */
 public class H2Engine extends AbstractDatabaseEngine {
+    /**
+     * The max length of a VARCHAR type.
+     */
+    private static final int MAX_VARCHAR_LENGTH = 1048576;
```

Comment:
Question: Why this value?

Reply:
That value comes from the [documentation](http://www.h2database.com/html/datatypes.html#:~:text=CHARACTER%20VARYING,-%7B%20%7B%20CHARACTER%20%7C%20CHAR&text=create%20a%20quote.-,The%20allowed%20length%20is%20from%201%20to%201048576%20characters.,when%20using%20this%20data%20type.)

Reply:
Alright! I would add that link on the javadoc for reference :)